drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -28,6 +28,16 @@ class Hmis::Hud::Client < Hmis::Hud::Base
 
   validates_with Hmis::Hud::Validators::ClientValidator
 
+  # ! Elliot: This logic probably wouldn't live here, but I've included it here for reference
```

Comment:
@eanders needs to setup
Client has_one client_headshot (ClientFile) with appropriate tag.

@ZachCMP can you add generic file upload backed by ClientFile

Reply:
This should look something like `set_local_client_image_cache`

```
file = GrdaWarehouse::ClientFile.create(
          client_id: id,
          user_id: user.id,
          name: 'Client Headshot',
          visible_in_window: false,
        )
file.tag_list.add('Client Headshot')
        file.client_file.attach(io: StringIO.open(image_data), filename: "client_headshot_#{id}")
```